server/tests/endpoints/certification/test_certification.py

```diff
@@ -504,30 +567,7 @@ def test_bios_is_none(generator: DataGenerator, test_client: TestClient):
     )
 
     assert response.status_code == 200
-    assert response.json() == {
-        "status": "Certified",
-        "architecture": "amd64",
-        "bios": None,
-        "board": {
-            "manufacturer": "Known Vendor",
-            "product_name": "Matching Board",
-            "version": "v1.0",
-        },
-        "chassis": None,
-        "available_releases": [
-            {
-                "distributor": "Ubuntu",
-                "version": release.release,
-                "codename": release.codename,
-                "kernel": {
-                    "name": report.kernel.name,
-                    "version": report.kernel.version,
-                    "signature": report.kernel.signature,
-                    "loaded_modules": [],
-                },
-            }
-        ],
-    }
```

Comment:
Is there a particular reason for modifying the test to check only for the field "status" being "Certified"? It seems like we're weakening the test this way. For instance if some code change causes other json changes then this test will no longer catch it. I'd propose we always make sure the whole output is correct. We can avoid making this check every time by using the same helper class we defined earlier that generated the status request. We can add three methods to this class that takes the output returned and checks if it's the same as the expected output (that it knows because it generated the status request initially). These methods would be something like check certified, check partial certified and check not certified. The reason for having three methods is to avoid adding logic to decide whether it's certified or not.